public/app/features/scenes/scenes/nestedGrid.tsx

```diff
@@ -0,0 +1,131 @@
+import { getDefaultTimeRange } from '@grafana/data';
+
+import { Scene } from '../components/Scene';
+import { SceneTimePicker } from '../components/SceneTimePicker';
+import { VizPanel } from '../components/VizPanel';
+import { SceneFlexChild, SceneFlexLayout } from '../components/layout/SceneFlexLayout';
+import { SceneGridCell, SceneGridLayout, SceneGridRow } from '../components/layout/SceneGridLayout';
+import { SceneTimeRange } from '../core/SceneTimeRange';
+import { SceneEditManager } from '../editor/SceneEditManager';
+import { SceneQueryRunner } from '../querying/SceneQueryRunner';
+
+export function getGridNestedTest(): Scene {
+  const panel = new VizPanel({
+    pluginId: 'timeseries',
+    title: 'Fill height',
+  });
+
+  const row = new SceneGridRow({
+    title: 'Nested grid layout',
+    size: { x: 0, y: 0, height: 11 },
+    children: [
+      new SceneGridLayout({
+        children: [
+          new SceneGridCell({
+            size: { x: 0, y: 0, width: 12, height: 10 },
+            isResizable: true,
+            isDraggable: true,
+            children: [
+              new VizPanel({
+                pluginId: 'timeseries',
+                title: 'No resize/no drag',
+              }),
+            ],
+          }),
+          new SceneGridCell({
+            size: { x: 12, y: 0, width: 12, height: 5 },
+            isResizable: true,
+            isDraggable: true,
+            children: [
+              new VizPanel({
+                pluginId: 'timeseries',
+                title: 'No resize/no drag',
+              }),
+            ],
+          }),
+          new SceneGridCell({
+            size: { x: 12, y: 5, width: 12, height: 5 },
+            isResizable: true,
+            isDraggable: true,
+            children: [
+              new VizPanel({
+                pluginId: 'timeseries',
+                title: 'No resize/no drag',
+              }),
+            ],
+          }),
+        ],
+      }),
+    ],
+  });
+  const cell1 = new SceneGridCell({
+    isResizable: true,
+    isDraggable: true,
+    size: {
+      x: 0,
+      y: 11,
+      width: 12,
+      height: 10,
+    },
+    children: [panel],
```

Comment:
having the same visible scene object instance in multiple places in the object graph is not recommended,  it could breaks the active / deactivate state, and editor selected object state.